src/html/tags/HtmlTagA.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import { View } from 'react-native';
+import { Text } from 'react-native';
 
 import { Touchable } from '../../common';
 import renderHtmlChildren from '../renderHtmlChildren';
 
-export default ({ onPress, href, ...restProps }) => (
+export default ({ onPress, href, cascadingChildStyle, ...restProps }) => (
   <Touchable onPress={() => onPress(href)} >
-    <View>
+    <Text style={[cascadingChildStyle]}>
```

Comment:
One of the main goals of the refactoring I did last time was to make sure we do not use `Text` anywhere except in the last children. Otherwise, it produces a crash.